src/styles/blocks/card.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $main-card-width + 100px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding-top: 32px;
+
+  font-family: 'Roboto ', sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+
+  transition: $main-transition;
+
+  &__img {
+    width: $width-img;
+    height: $hight-img;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    width: $width-content;
+    font-size: 1.2em;
+    font-weight: 500;
+    line-height: 18px;
```

Comment:
The font-size should be defined in 'px', 'em', or 'rem' units according to the code style rules, rather than using 'em' units directly. This ensures consistency across different browsers and devices.